packages/atomic/src/components.d.ts

```diff
@@ -2535,12 +2536,12 @@ declare global {
         new (): HTMLAtomicAriaLiveElement;
     };
     /**
-     * @beta - This component is part of the automatic facets feature.
-     * Automatic facets are currently in beta testing and should be available soon.
-     * An automatic facet is a special type of facet generated by the automatic facets feature. Unlike traditional facets that
-     * need to be explicitly defined and requested in the search query, automatic facets are dynamically generated by the index
-     * in response to the search query. It's important to note that this component should not be used directly. It is internally used
-     * by the `atomic-automatic-facet-generator` component to automatically render updated facets.
+     * An automatic facet is a special type of facet generated by the automatic facets feature.
```

Comment:
Whoops, just realized now that I'm commenting in the generated code 🤦 Either way, be sure to apply my suggestions (if you agree) universally in their respective places as it appears that it was re-used throughout 👍